packages/vm/examples/l2-playground/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "scripts": {
+    "fetch": "ts-node fetch-polygon.ts"
+  },
+  "devDependencies": {
+    "node-fetch": "3.2.0"
+  }
+}
```

Comment:
Why are you now introducing a `package.json` file here? I thought you would have just removed these and put eventual dependencies in the main `devDependencies`? 🤔

Reply:
Because the develop branch does not have the latest changes from master, which is this new structure you're talking about. I thought that in order to keep consistency in the code, for now, to use the same structure we currently have, and then migrate to the latest one when the time of merge comes. Also, this helped me to have the script in the `package.json` and run it easier with the debugger from WebStorm 😛